Align global norm test sharding

liuzhenhai93 opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->

### PR changes
<!-- One of [ OPs | APIs | Docs | Others ] -->

### Description
<!-- Describe what you’ve done -->

Commit:
polish

Commit:
polish

Commit:
polish

Commit:
polish

Commit:
format

Commit:
polish

Commit:
polish

liuzhenhai93 committed:
test sharding

liuzhenhai93 committed:
polish

liuzhenhai93 committed:
polish

liuzhenhai93 committed:
polish

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/Paddle?pullRequest=55341) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/Paddle?pullRequest=55341) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: liuzhenhai93<br/>:x: Your Name<br/><hr/>**Your Name** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/Paddle?pullRequest=55341) it.</sub>

liuzhenhai93 committed:
format

liuzhenhai93 committed:
polish

liuzhenhai93 committed:
parent 11c26f26d3437438264452fd3121b0c2cc9a9f8b
author liuzhenhai93 <liuzhenhai93@outlook.com> 1689137331 +0000
committer liuzhenhai93 <liuzhenhai93@outlook.com> 1689137377 +0000

polish

format

polish

liuzhenhai93 committed:
Merge branch 'align_global_norm_test_sharding' of https://github.com/liuzhenhai93/Paddle into align_global_norm_test_sharding
